imports/plugins/core/files/server/no-meteor/jobs/saveTempImages.js

```diff
@@ -0,0 +1,38 @@
+import { Jobs } from "/imports/utils/jobs";
+
+/**
+ * @method saveTempImages
+ * @summary Processes the Jobs to convert an remote image
+ * @param {Object} fileWorker - The created instance of TempFileStoreWorker
+ * @return {undefined}
+ */
+export default function saveTempImages(fileWorker) {
+  const saveImagesJob = Jobs.processJobs("saveImage/local", {
+    pollInterval: 60 * 60 * 1000, // Retry failed images after an hour
```

Comment:
It would make sense. Are you thinking ENV variables or app settings with a Reaction Admin UI to set them? I'll have to think about it, but we might want to consider this more of a "jobs" thing than a "files" thing. I could figure out a solution to make these configurable for all types of jobs when doing the jobs-collection PR.

Reply:
If it's an overall `jobs` thing, I'd say put it in the UI since it touches many things and might need to be adjusted a few times easily to get the right setup. If it's just for `files` here, then I'm torn between using `env` or putting it in the UI.

UI:
- It's not a secret, so does't _need_ to be hidden in the `env`
- If these need to be changed, do we really need to force a restart to reload the `env`

ENV:
- These probably won't be changed more than once, if ever, so putting them in the UI might be overkill

Reply:
OK, I put this in my notes for the jobs project. I think it's best to come up with a solution there, but it's hard to know what it should be until I get into it. I agree UI seems slightly better choice.